_posts/2021-01-04-dynamic-test-oracles-for-rho-problems.html

```diff
@@ -253,4 +253,79 @@ <h2 id="comments-header">
 		</div>
 		<div class="comment-date">2021-01-06 23:01 UTC</div>
 	</div>
+	
+	<div class="comment" id="4aa4188124ca4f4fadf35d6881b9452e">
+		<div class="comment-author"><a href="https://about.me/tysonwilliams">Tyson Williams</a></div>
+		<div class="comment-content">
+		<p>
+			A thought provoking post indeed!
+		</p>
+		<blockquote>
+			In F# I'm going to use <a href="https://fscheck.github.io/FsCheck">FsCheck</a>...
+		</blockquote>
+		<p>
+			I think that is a fine choice given the use case laid out in this post.  In general though, I think <a href="https://github.com/hedgehogqa/fsharp-hedgehog">Hedgehog</a> is a better property-based testing library.  Its killer feature is integrated shrinking, which means that all generators can also shrink and this extra power is essentially free.
+		</p>
+		<p>
+			For the record (because this can be a point of confusion), Haskell has <a href="https://hackage.haskell.org/package/QuickCheck">QuickCheck</a> and <a href="https://hackage.haskell.org/package/hedgehog">(Haskell) Hedgehog</a> while F# has ports from Haskell called <a href="https://fscheck.github.io/FsCheck">FsCheck</a> and <a href="https://github.com/hedgehogqa/fsharp-hedgehog"></a>(FSharp) Haskell</a>.
```

Comment:
There's something curious going on here with the link mark-up towards the end. I'd fix it if I could intuit your intent, but alas, I can't.

Reply:
Oh, thanks for the help!

I think I found an extra `</a>`.  I pushed a commit that removes it.  Does that look right now?